src/stores/video.ts

```diff
@@ -469,12 +505,20 @@ export const useVideoStore = defineStore('video', () => {
 
       const chunkBlobs = chunks.map((chunk) => chunk.blob)
       debouncedUpdateFileProgress(info.fileName, 50, 'Processing video chunks.')
-      const durFixedBlob = await fixWebmDuration(new Blob([...chunkBlobs], { type: 'video/webm;codecs=vp9' }))
+
+      const mergedBlob = new Blob([...chunkBlobs], { type: 'video/webm;codecs=vp9' })
+
+      let durFixedBlob: Blob | undefined = undefined
+      try {
+        durFixedBlob = await fixWebmDuration(mergedBlob)
+      } catch {
+        Swal.fire({ text: 'Failed to fix the video duration. The recorded video may present issues.', icon: 'error' })
```

Comment:
This Swal also could be replaced by our interactionDialog. This would probably fix the video modal closing issue that was pointed on the following comment:

![image](https://github.com/bluerobotics/cockpit/assets/14910201/176085e4-4521-4fc8-8fcf-3199774dbfac)

Reply:
Forgot about this one!